department-of-veterans-affairs/caseflow
# Remove duplicate word hearing in update alert

Resolves #14991

### Description
Remove duplicate word hearing in progress alert

### Acceptance Criteria
- [ ]  Progress alert should not include the word hearing twice 

### Testing Plan
1. Switch to User BVASYELLOW (VACO)
2. Select Hearings Tab
3. Select a hearing with type 'Central'
4. Click 'Edit Hearing Details'
5. Change the hearing type to 'Virtual'
6. On the 'Convert to Virtual Hearing' page, select the appellant or POA timezone option, and select a new timezone
9. Click 'Save'
10. Validate that the in progress alert no longer includes the word 'hearing' twice

### User Facing Changes

 BEFORE|AFTER
 ---|---
![d58e4788-7711-4cf2-8d8a-cd76b4d4cfa3](https://user-images.githubusercontent.com/68879427/91620299-6634b680-e944-11ea-9942-ff5b91a1c5be.png) |
